src/fides/api/task/graph_task.py

```diff
@@ -584,57 +586,69 @@ def access_request(self, *inputs: List[Row]) -> List[Row]:
     def erasure_request(
         self,
         retrieved_data: List[Row],
-        inputs: List[List[Row]],
```

Comment:
Nothing was actually using these inputs so we've removed it.  It was for a previous version of the Email Connector that was later replaced.  Removing it prevents us from having to manage this same concept in DSR 3.0 

https://github.com/ethyca/fidesops/pull/1168